python/test/test_polars.py

```diff
@@ -94,19 +94,22 @@ def test_expr_ffi():
     assert str(t_ident((lf, pl.col("A")))[1]) == str(pl.col("A"))
 
 
-def test_private_lazyframe_explicit_sum():
+@pytest.mark.parametrize(
+    "measure", [dp.max_divergence(T=float), dp.zero_concentrated_divergence(T=float)]
+)
+def test_private_lazyframe_explicit_sum(measure):
     pl = pytest.importorskip("polars")
     pl_testing = pytest.importorskip("polars.testing")
 
     lf_domain, lf = example_lf(
         margin=["B"], public_info="keys", max_partition_length=50
     )
 
-    expr = pl.col("A").fill_null(0.0).clip(0.0, 1.0).sum().dp.laplace(0.0)
+    expr = pl.col("A").fill_null(0.0).clip(0.0, 1.0).sum().dp.noise(0.0)
```

Comment:
What determines which kind of noise is applied? Is the idea that we'll know from context whether gaussian or laplace is appropriate?

Reply:
You know based on the privacy measure!